.vsts-ci/mac.yml

```diff
@@ -66,12 +67,12 @@ jobs:
     condition: succeeded()
 
   - powershell: |
-      tools/travis.ps1 -Stage Failure
+      Write-Host "Tests failed."
     displayName: After Failure
     condition: failed()
 
   - powershell: |
-      tools/travis.ps1 -Stage Success
+      Write-Host "Tests did not fail!"
```

Comment:
I think you could probably just log success or fail inside of the psm1 and then you can safely take out this task.

Reply:
Tyler is right, this design was due to some bugs in Travis-CI

Reply:
I moved it to the PSM1.